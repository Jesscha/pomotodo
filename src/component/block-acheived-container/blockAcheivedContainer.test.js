import React from 'react';
import { shallow } from 'enzyme';
import {BlockAcheivedContainer} from './blockAcheivedContainer.component';
import Pomoblock from '../pomoblock/pomoblock.component';


describe("<BlockAcheivedContainer />", ()=>{
    const wrapper = shallow(<BlockAcheivedContainer  achievedBlocks={3}/>)
   


    it("renders BlockAcheivedContainer", ()=>{
        expect(wrapper).toMatchSnapshot();
    }); 

    

    it("renders given amount of PomoBlocks", ()=>{
        expect(wrapper.find(Pomoblock).length).toEqual(3);
    }); 

    
    })





