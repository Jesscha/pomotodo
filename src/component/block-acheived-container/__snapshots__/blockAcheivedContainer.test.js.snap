// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlockAcheivedContainer /> renders BlockAcheivedContainer 1`] = `ShallowWrapper {}`;
